servers/lib/.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    project: 'tsconfig.json',
+    tsconfigRootDir: __dirname,
+    sourceType: 'module',
+  },
+  plugins: ['@typescript-eslint/eslint-plugin'],
+  extends: [
+    'plugin:@typescript-eslint/recommended',
+    'plugin:prettier/recommended',
+  ],
+  root: true,
+  env: {
+    node: true,
+    jest: true,
+  },
+  ignorePatterns: ['.eslintrc.js'],
+  rules: {
+    '@typescript-eslint/interface-name-prefix': 'off',
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/explicit-module-boundary-types': 'off',
+    '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
By turning these rules off it's like forcing eslint to not format for TypeScript and allow javaScript, which would be kind of redundant to even use TypeScript then.
You are specifically turning off return type declaration in line 21. Why use TypeScript then?

Reply:
I had not noticed that, ive made the change, so that the are all set to 'on', thanks!

Reply:
We just had a meeting with Prasad, he suggested that our group and you plan a meeting to look at getting common config and script files, so maybe let this be standby for now until then.

Reply:
Sounds like a good idea - Do you have time monday morning/noon?